config/settings/base.py

```diff
@@ -23,7 +23,7 @@
 PROJECT_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 BASE_DIR = os.path.dirname(PROJECT_DIR)
 
-DEBUG = strtobool(os.getenv("DEBUG", "False"))
+DEBUG = strtobool(os.getenv("DEBUG", "True"))
```

Comment:
Default DEBUG must be False, otherwise if this env variable is removed, user will get to view error log.